backend/tests/apps/sitemaps/sitemaps_test.py

```
@@ -0,0 +1,193 @@
+class TestCachedSitemapView(unittest.TestCase):
+    def test_cached_sitemap_view_returns_view(self):
+        sitemaps = {"static": sm.StaticSitemap()}
+        view = sm.cached_sitemap_view(sitemaps)
+        assert callable(view)
+        factory = RequestFactory()
+        request = factory.get("/sitemap.xml")
+        with patch("apps.sitemaps.sitemaps.sitemap") as mock_sitemap:
+            mock_sitemap.return_value = HttpResponse("ok")
+            response = view(request)
+            assert response.status_code == 200
+            assert response.content == b"ok"
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)